src/constants/features.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/prefer-default-export
+const featuresMapping = {
+    polygonSupportGeo: 'polygon_support_geo',
+    zoomableImage: 'zoomable_image',
+    privateProject: 'privateProject',
```

Comment:
The feature names from server are usually snake_case. This looks problematic

Reply:
It was a test. You passed!